client/verta/tests/test_entities.py

```diff
@@ -604,6 +604,17 @@ def test_find_by_tag(self, client):
         found_runs = expt.expt_runs.find("tags ~= {}".format(tag))
         assert len(found_runs) == len(runs_with_tag)
 
+        runs_with_tag[-1].log_hyperparameter("some-hyper", 1)
+        # compound conditions:
+        assert len(expt.expt_runs.find(["tags ~= {}".format(tag), "hyperparameters.some-hyper == 1"])) == 1  # old syntax
+        assert len(expt.expt_runs.find("tags ~= {}".format(tag), "hyperparameters.some-hyper == 1")) == 1  # new syntax
+
+        # if any predicate is not string, should fail:
+        with pytest.raises(TypeError) as excinfo:
+            expt.expt_runs.find("tag ~= {}".format(tag), 1234)
+
+        assert "predicates must all be strings" in str(excinfo.value).strip()
```

Comment:
Learned this little trick from Evgenii!
```suggestion
        with pytest.raises(TypeError, match="predicates must all be strings"):
            expt.expt_runs.find("tag ~= {}".format(tag), 1234)
```